scrunch/datasets.py

```diff
@@ -1879,7 +1879,10 @@ def __iter__(self):
         """
         Iterable of subvariable aliases
         """
-        for svar in self.resource.subvariables.index.values():
+        values = self.resource.subvariables.index.values()
+        sort_func = lambda x: x['id']
+
+        for svar in sorted(values, key=sort_func):
```

Comment:
I don't think that the intention of the ticket was to use a sorting mechanism. Instead we should be trying to serialize/de-serialize the objects in using ordered objects. We will just have to trust that on the other end of the serialize/de-serialize json dance they are doing the same.